src/status_im/contexts/wallet/connected_dapps/scan_dapp/view.cljs

```diff
@@ -0,0 +1,43 @@
+(ns status-im.contexts.wallet.connected-dapps.scan-dapp.view
+  (:require [quo.core :as quo]
+            [quo.foundations.resources :as quo.resources]
+            [quo.theme]
+            [react-native.core :as rn]
+            [react-native.hooks :as hooks]
+            [status-im.common.scan-qr-code.view :as scan-qr-code]
+            [status-im.contexts.wallet.connected-dapps.scan-dapp.style :as style]
+            [utils.debounce :as debounce]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [{:keys [keyboard-shown]}   (hooks/use-keyboard)
+        {:keys [name emoji color]} (rf/sub [:wallet/current-viewing-account])]
+    [:<>
+     (when keyboard-shown
+       (rn/dismiss-keyboard!))
```

Comment:
best to use the `use-effect` hook here, otherwise this will run every time the component re-renders

Reply:
but `use-effect` is deprecated now

Reply:
We can also use `use-mount`, but since we have the keyboard-shown, dependency, we should go with `use-effect`. This deprecation still seems weird, as it's an essential feature of react.

Reply:
`use-mount` uses `use-effect` anyway — have no idea why and who went with this sort of guideline but I see a some reasoning behind not using `use-effect` in re-frame codebase, kinda agree with that but find the chosen solution at least weird.

I put it inside `use-mount` in the last commit.